File: matnam/src/main/java/com/grepp/matnam/app/model/kafka/dto/SmtpDto.java
package com.grepp.matnam.app.model.kafka.dto;

import java.util.List;
import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SmtpDto {
    private String from;
    private String subject;
    private String to;
    private Map<String, String> properties;
    private String templatePath;
    private String eventType;
}

File: NBE5-6-3-Team06-Mail/src/main/kotlin/com/grepp/spring/infra/mail/SmtpDto.kt
package com.grepp.spring.infra.mail

import com.fasterxml.jackson.annotation.JsonTypeName

@JsonTypeName("SmtpDto")
data class SmtpDto(
    val from:String,
    val subject:String,
    val to:String,
    val properties:Map<String, Any>? = mutableMapOf(),
    val templatePath:String,
    val eventType: String
)